tests/functional/Glpi/Inventory/Inventory.php

```diff
@@ -211,7 +211,7 @@ private function checkComputer1($computers_id)
             [
                 'logical_number' => 0,
                 'name' => 'lo',
-                'instantiation_type' => 'NetworkPortEthernet',
```

Comment:
Will not that change impact existing inventories?

Reply:
I guess it will not be possible to play a data update to get those kind of wrong type issues fixed; but I wonder f we will not be creating duplicates in network ports. If old port i properly replaced, that's probably not a big deal.